### PR TITLE
[PE-4639] Fix accessibility warnings in Brand component

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/PE-4639

- Removed sr-only which caused contrast errors (The other solution would be to set a -d--none class)
- Added role and aria-labelledby attrs and title tag for accessibility passing in SVG's.

### CODE REVIEW

File: src/website/views/templates/error-404.pug [hunk 1]
```diff
@@ -9,7 +9,8 @@ description: Basic 404 level error page template. Indicates a client error, such
       header.chi-header.-inverse
         .chi-header__content.-justify-content--center
           a.chi-brand(href="#", aria-label="CenturyLink")
-            svg(viewBox='0 0 204 41', xmlns='http://www.w3.org/2000/svg')
+            svg(viewBox='0 0 204 41', xmlns='http://www.w3.org/2000/svg', aria-labelledby="CenturyLink", role="img")
```
Comment (janibolkvadze) [inline src/website/views/templates/error-404.pug:12]:
```suggestion
            svg(viewBox='0 0 204 41', xmlns='http://www.w3.org/2000/svg', aria-label="CenturyLink", role="img")
```

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://2c97f6867bb4b0ca8d2d40ddd22c90f800bc7fe3-chi-pr.enterprise.ctl.io" target="_blank"> https://2c97f6867bb4b0ca8d2d40ddd22c90f800bc7fe3-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/863/ for more details.

File: src/website/layouts/partials/header.pug [hunk 2]
```diff
@@ -43,8 +43,17 @@ header.chi-header.-inverse.docs-header.-z--20
         .chi-btn__content
           i.chi-icon.-sm--2.icon-menu
       a.chi-brand.docs-header__logo(href=`${rootPath}`, aria-label='CenturyLink')
-        img(src='https://assets.ctl.io/images/centurylink-logo-white.svg', alt='CenturyLink')
-        span.-sr--only CenturyLink
+        svg(viewBox='0 0 204 41', xmlns='http://www.w3.org/2000/svg', aria-labelledby='CenturyLink', role='img')
```
Comment (mattnickles) [inline src/website/layouts/partials/header.pug:46]: `aria-labeledby` is not used correctly. It's value should be [one or more element IDs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute). From mozilla developer docs: "This idea is similar to how the "for" attribute is used to link a LABEL to an INPUT."

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://6bd1165c6e0cb61ce2d0c5600a4712a2636d1dd1-chi-pr.enterprise.ctl.io" target="_blank"> https://6bd1165c6e0cb61ce2d0c5600a4712a2636d1dd1-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/880/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-pioguo, https://ops.elasticbox.com/services/instances/i-r1mgtc

**Pull request merged**